hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/dml/TestInsertTable.scala

```diff
@@ -1056,6 +1056,50 @@ class TestInsertTable extends HoodieSparkSqlTestBase {
     }
   }
 
+  test("Test combine before bulk insert") {
```

Comment:
can we cover all combinations of `hoodie.combine.before.insert` and `hoodie.populate.meta.fields`?

Reply:
It's a good point, I will extend test cases.

Reply:
Added check of all combinations of `hoodie.combine.before.insert` and `hoodie.populate.meta.fields` in the unit test.